pkg/kev/skaffold_test.go

```diff
@@ -136,4 +136,57 @@ var _ = Describe("Skaffold", func() {
 			}))
 		})
 	})
+
+	Describe("UpdateProfiles", func() {
+		var manifest *kev.SkaffoldManifest
+
+		envName := "testenv"
+
+		BeforeEach(func() {
+			envs := []string{envName}
+			manifest = kev.BaseSkaffoldManifest()
+			manifest.SetProfiles(envs)
+		})
+
+		Context("for skaffold profile names matching rendereded environment", func() {
+
+			When("rendered manifests output path is a directory", func() {
+				outputPath := "testdata" // point at any existing directory for test!
+
+				envToOutputPath := map[string]string{
+					envName: outputPath,
+				}
+
+				It("updates the matching profile with new manifests path selecting all the files in that directory", func() {
+					manifest.UpdateProfiles(envToOutputPath)
+					Expect(manifest.Profiles[0].Deploy.KubectlDeploy.Manifests).To(ContainElement(filepath.Join(outputPath, "*")))
+				})
```

Comment:
It would be great if `manifest` has a convenience method to return the value in `manifest.Profiles[0].Deploy.KubectlDeploy.Manifests`.

Reply:
there is currently no need for it but might add some as and when required.